dev/MAINTAINING.md

```diff
@@ -138,8 +138,12 @@
           `extra-source-files`, these wild-cards may match additional
           files lying around in the workspace and unintentionally ship
           them as well.
-          * `cabal v2-sdist`; `cabal upload --publish <tarpath>`
-          * `stack upload .`
+          * Upload and verify the candidate
+            * `cabal v2-sdist; cabal upload <tarpath>`
+            * `cabal v2-haddock --haddock-for-hackage --enable-doc; cabal upload -d <tarpath>`
+          * Publish the package
+            * `cabal v2-sdist`; `cabal upload --publish <tarpath>`
+        * `stack upload .`
```

Comment:
This is not equivalent to the cabal workflow, so let's remove it.

Reply:
Where do we move it to?
This is on the same level as `Upload to hackage`. It isn't coming under the cabal workflow.

Reply:
Remove it.

Reply:
Delete this line.